components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/java/org/wso2/carbon/identity/api/server/idp/v1/core/ServerIdpManagementService.java

```diff
@@ -1953,6 +1953,12 @@ private void processPatchRequest(List<Patch> patchRequest, IdentityProvider idpT
                     case Constants.HOME_REALM_PATH:
                         idpToUpdate.setHomeRealmId(value);
                         break;
+                    case Constants.ALIAS:
+                        idpToUpdate.setAlias(value);
+                        break;
+                    case Constants.CERTIFICATE:
+                        idpToUpdate.setCertificate(value);
```

Comment:
if its a certificate it will be an object. so we need to do something similar to https://github.com/wso2/identity-api-server/blob/master/components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/java/org/wso2/carbon/identity/api/server/idp/v1/core/ServerIdpManagementService.java#L1136-L1141

Reply:
Fixed https://github.com/wso2/identity-api-server/pull/114/commits/8382cd79bb7b201e3bf5e9c66077c0b748be4bde